e2e/cypress/integration/interactive_menu/basic_options_spec.js

```diff
@@ -189,6 +189,37 @@ describe('Interactive Menu', () => {
             });
         });
     });
+
+    it('IM21037 - Clicking in / Tapping on the message attachment menu box opens list of selections', () => {
+        // # Create a message attachment with menu
+        cy.postIncomingWebhook({url: incomingWebhook.url, data: payload});
```

Comment:
It's failing the test due to unexpected payload.  Looks like it's being mutated.  I suggest to create new payload for incoming webhook.
```suggestion
        const basicOptionPayload = getMessageMenusPayload({options});
        cy.postIncomingWebhook({url: incomingWebhook.url, data: basicOptionPayload});
```